abs.yaml

```diff
@@ -0,0 +1 @@
+aggregate_branch: 3.12
```

Comment:
Remove this and it will fix your build issue. You added the submodule in master and not in 3.12. But you don't need the 3.12 aggregate branch for this feedstock.